Makefile

```diff
@@ -404,13 +404,21 @@ ifneq ($(ECK_IMAGE),)
 else
 	$(MAKE) docker-build
 endif
-kind-e2e: export KUBECONFIG = ${HOME}/.kube/kind-config-eck-e2e
-kind-e2e: export NODE_IMAGE = ${KIND_NODE_IMAGE}
-kind-e2e: kind-node-variable-check set-kind-e2e-image dep-vendor-only e2e-docker-build
+
+kind-e2e: kind-e2e-cluster kind-e2e-run
+
+kind-e2e-cluster: export NODE_IMAGE = ${KIND_NODE_IMAGE}
```

Comment:
The idea is to use some specific images, for different versions of K8S, with all the Elasticsearch/Kibana/APM images pre-loaded to avoid to download them (see https://github.com/kubernetes-sigs/kind/pull/777).
I'm not sure if there's a "default" image.